code/model/model.h

```diff
@@ -23,6 +23,8 @@
 #include "object/object.h"
 #include "ship/ship_flags.h"
 
+#include <set>
+
```

Comment:
This shouldn't be #included; we have `SCP_` equivalents of most collections.  See comments below.